src/main/resources/application.conf

```diff
@@ -12,65 +12,29 @@ kamon {
 
   show-aspectj-missing-warning = on
 
-  metric {
-    filters {
-      # Define filter for your actor names here
-      # Example 1: akka-actor.includes = [ "default/user/**" ]
-      # Example 2:
-      # akka-actor {
-      # includes = [ "my-app/user/job-manager", "my-app/user/worker-*" ]
-      # excludes = [ "my-app/system/**", "my-app/user/worker-helper" ]
```

Comment:
We had an exclude example here.  Does Kamon 1.x not have support for that?  If it has it, should we add example(s) of that, too?

Reply:
Yes. Kamon supports `exclude` also. Will add an example